README.md

```diff
@@ -1,6 +1,20 @@
 Absolute
-=======
+=============
+
+### A web-based order platform like siren-order for Starbucks.
```

Comment:
I'd like to present our project with one sentence actually. :-) I agree with you, many famous github projects explain what they are with one or short sentences so but not like us, I think we also need a good words to show us. anyway.